programs/editor/src-bellabello.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```

Comment:
What's the reasoning behind the separate control? I.e., why can't you simply load the plugin into the src-variant of the split screen editor?